core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/Submitting.java

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.kotlinx.dataframe.api;
+
+import kotlin.Unit;
+
+import java.io.File;
+
+public class Submitting {
+
+    public boolean sample() {
+        int number = 1;
```

Comment:
what is this?

Reply:
uncommitted temp file, will be removed :)